gulpfile.js

```diff
@@ -286,13 +285,13 @@
 			const CLI_VERSION = require(path.join(CLI_PATH, "package.json")).version;
 			const CONTROLS_PATH = path.join(__dirname, "node_modules", "@chartiq", "finsemble-react-controls");
 			const CONTROLS_VERSION = require(path.join(CONTROLS_PATH, "package.json")).version;
-			
-			// Check e2o version
+
+			// Check electron adpater version
 			const E2O_PATH = path.join(__dirname, "node_modules", "@chartiq", "e2o");
```

Comment:
Do we want to leave `E2O_PATH` and `E2O_EXISTS` until we rename the package?

Reply:
yeah